src/components/ActionsBuilder/SupportedActions/RawTransaction/RawTransactionEditor.tsx

```diff
@@ -0,0 +1,158 @@
+import { Button } from 'components/primitives/Button';
+import { AddressInput } from 'components/primitives/Forms/AddressInput';
+import {
+  Control,
+  ControlLabel,
+  ControlRow,
+} from 'components/primitives/Forms/Control';
+import { ErrorLabel } from 'components/primitives/Forms/ErrorLabel';
+import { TokenAmountInput } from 'components/primitives/Forms/TokenAmountInput';
+import { BigNumber } from 'ethers';
+import { useMemo } from 'react';
+import { Controller, useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { ActionEditorProps } from 'components/ActionsBuilder/SupportedActions';
+import validateRawTransaction from './validateRawTransaction';
+import { RawTransactionValues } from './types';
+import { TextArea } from 'components/primitives/Forms/TextArea';
+
+const RawTransactionEditor: React.FC<ActionEditorProps> = ({
+  decodedCall,
+  onSubmit,
+}) => {
+  const parsedData = useMemo(() => {
+    return {
+      to: decodedCall?.to,
+      value: decodedCall?.value,
+      data: decodedCall?.optionalProps?.data,
+    };
+  }, [decodedCall]);
+
+  const { t } = useTranslation();
+
+  const { control, handleSubmit } = useForm({
+    resolver: validateRawTransaction,
+    context: { t },
+    defaultValues: {
+      to: parsedData?.to,
+      value: parsedData?.value,
+      data: parsedData?.data,
+    },
+  });
+
+  const submitAction = (values: RawTransactionValues) => {
+    if (!values.data) values.data = '0x00';
+    if (!values.value) values.value = BigNumber.from(0);

```

Comment:
I think you can remove these lines and simply do `value: BigNumber.from(values.value ?? 0)` on submit or something like that. 

Not a good practice to reassign param value.
https://eslint.org/docs/latest/rules/no-param-reassign

Reply:
Fixed! Also, I need to have more present the bullish coalescing operator.

For the `data` part, I went with the `||` because I needed to assign any falsy value, as the component returns an empty string.